apps/admin/traefik2/demo/traefik-health-endpoint.yaml

```diff
@@ -0,0 +1,32 @@
+---
+kind: Service
```

Comment:
can the service be reviewed? There is already a traefik service in place on admin namespace

```
kubectl get svc                     
NAME      TYPE         
traefik   LoadBalancer
```

If this is needed, you need to add a patch for the svc